CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 ## 15.0.0 / unreleased
 
 * [FEATURE] Drop Ruby < 3.1 support
+* [ENHANCEMENT] Route helper locale parameters now work without host_locales configuration ([#329](https://github.com/enriclluelles/route_translator/pull/329))
+
+  Route helpers now respect locale parameters (e.g., `posts_path(locale: "en")`) regardless
+  of host_locales configuration. Previously these were silently ignored.
+
+  Workarounds like `I18n.with_locale(:en) { posts_path }` can be replaced with 
+  the now-working `posts_path(locale: "en")` syntax.
```

Comment:
Please here just a single, one line description of the change

Then also add POTENTIAL BREAKING CHANGE

```
* [BUGFIX] POTENTIAL BREAKING CHANGE: ... 
```

Please add an entry in the `UPGRADING.md` guide with the rest of this description